### DISCUSSION TITLE
Miota auf 0 shimmer und assembly zu wenig

### DISCUSSION THREAD

**Discussion by Schoko1984** [iotaledger/firefly#3011] [Help & Support]
Habe eine Frage. Seit dem Update der firefly wallet sehe ich meine miota nicht mehr und die Anzeige für shimmer und Assembly token zeigt zu wenig an was kann ich tun ???

**Comment by Schoko1984** [comment]


**Reply by Dr-Electron** [reply]
Dein neue Addresse ist keine gültige iota addresse sondern eine fürs devnet. Was hast du damit gemacht? IOTAS hingesendet? Lässt Firefly das überhaubt zu?

**Reply by Dr-Electron** [reply]
Und die addressen würde ich hier löschen.  Muss ja keiner wissen wieviel du besitzt, oder?

**Comment by Schoko1984** [comment]
wie devnet habe das update gemacht und jetzt steht die adresse da

**Reply by Dr-Electron** [reply]
Ja das ist ne devnet Adresse. Wahrscheinlich hast du das developer Profil aktiviert. Solltest du also einfach wieder umstellen können. Solange du da nichts hingesendet hast, sollte alles ok sein

**Comment by Schoko1984** [comment]
ok versuche ich danke

**Comment by Schoko1984** [comment]
wo soll das funktionieren

**Reply by Dr-Electron** [reply]
Ich bin nicht mehr vor einem PC, also kann ich dirs nicht genau sagen. Du kannst also entweder warten bis ich geschlafen habe :) oder einfach ein neues Profil ohne developer mode erstellen und da dein wallet mit deinen 24 wörtern, einem stronghold backup oder deinem ledger recovern
